config/default/rdfImporterConfig.conf.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return new oat\oatbox\config\ConfigurationService(array(
```

Comment:
I saw we already had a migration for this, but we forgot this in the installation: 

https://github.com/oat-sa/extension-tao-testtaker/blob/master/migrations/Version202105041023253148_taoTestTaker.php#L29

@mike-ionut-mihai-sandu-tao

Reply:
Yes, just rechecked, if we have the default file, it's is copied